discovery_engine_core/core/src/stack/filters.rs

```diff
@@ -116,4 +126,52 @@ mod tests {
             "Mensch mit d\u{00fc}sterer Prognose: \"Kollektiv versagt!\" N\u{00e4}chste Pandemie wird schlimmer als Covid-19",
         ]);
     }
+
+    #[test]
+    fn test_filter_title() {
+        let malformed_articles: Vec<Article> = serde_json::from_str(include_str!(
+            "../../test-fixtures/articles-invalid-title.json"
+        ))
+        .unwrap();
+        assert_eq!(2, malformed_articles.len());
+
+        let result = CommonFilter::apply(vec![].as_slice(), malformed_articles).unwrap();
```

Comment:
the empty slice should work too, similarly below.
```suggestion
        let result = CommonFilter::apply(&[], malformed_articles).unwrap();
```